main.css

```diff
@@ -16,6 +16,15 @@ main {
   font-family: 'Poppins', sans-serif;
 }
 
+.popup {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
[OPTIONAL] Well done on the popup 💯 this is optional as this was not required by the previous reviewer but this could really help improve your project, make sure that your design matches the Figma 🎨  take a look at the differences 👇🏽 

## Figma

![Screenshot 2023-04-19 at 08 12 48](https://user-images.githubusercontent.com/45739963/233071251-62cdc724-98fc-45d9-97a7-50343353537a.png)

## Your design (1)

![Screenshot 2023-04-19 at 08 15 09](https://user-images.githubusercontent.com/45739963/233071786-f3eb12cd-087d-4778-a2a2-eaa69a12affe.png)

## Your design (2)

![Screenshot 2023-04-19 at 08 15 45](https://user-images.githubusercontent.com/45739963/233071902-4a8f474e-0289-4d2e-8268-c6ef9062ff01.png)

Consider adding the right spacing (paddings and margins), adjust the font sizes, font family and the positioning of the elements (especially the ❌  button)